src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
BEM naming convention suggests that modifiers should be used with the block or element name. Instead of 'stopwatch--speed-up', it should be 'stopwatch stopwatch--speed-up' if 'speed-up' is a modifier for 'stopwatch'.